### PR TITLE
fixed widget docs issues, with missing descriptions and renamed a widget

### PR DESCRIPTION
**Pull request by alx365**
This only adds stuff to the docs (or the be generated ones :D), i almost had no description at any of the widgets i had added. I also changed a bit of your descriptions. 

This also  renames a widget, to what it actually is, namingly the combo box -> combo box text because that's what the code actually generates.

### CODE REVIEW

File: src/widgets/widget_definitions.rs [hunk 1]
```diff
@@ -247,7 +250,7 @@ fn build_gtk_scale(bargs: &mut BuilderArgs) -> Result<gtk::Scale> {
 }
 
 /// @widget input
-/// @desc an input field. For this to be useful, set `focusable="true"` on the window.
+/// @desc A input field. For this to be useful, set `focusable="true"` on the window.
```
Comment (elkowar) [inline src/widgets/widget_definitions.rs:253]:
```suggestion
/// @desc An input field. For this to be useful, set `focusable="true"` on the window.
```

File: src/widgets/widget_definitions.rs [hunk 2]
```diff
@@ -230,7 +233,7 @@ fn build_gtk_color_chooser(bargs: &mut BuilderArgs) -> Result<gtk::ColorChooserW
 }
 
 /// @widget scale extends range
-/// @desc a slider.
+/// @desc A widget with a slider.
```
Comment (elkowar) [inline src/widgets/widget_definitions.rs:236]: It is just simply the slider tho :o
```suggestion
/// @desc A slider.
```

File: src/widgets/widget_definitions.rs [hunk 3]
```diff
@@ -319,6 +324,7 @@ fn build_gtk_box(bargs: &mut BuilderArgs) -> Result<gtk::Box> {
 }
 
 /// @widget label
+/// @desc A widget that displays a small to medium amount of text
```
Comment (elkowar) [inline src/widgets/widget_definitions.rs:327]: Maybe explain why this is relevant if you can just use inline text too
```suggestion
/// @desc A text widget giving you more control over how the text is displayed
```

File: src/widgets/widget_definitions.rs [hunk 4]
```diff
@@ -289,6 +293,7 @@ fn build_gtk_button(bargs: &mut BuilderArgs) -> Result<gtk::Button> {
 }
 
 /// @widget image
+/// @desc A widget displaying images
```
Comment (elkowar) [inline src/widgets/widget_definitions.rs:296]: It only renders a single image 
```suggestion
/// @desc A widget displaying an image
```

File: src/widgets/widget_definitions.rs [hunk 5]
```diff
@@ -148,12 +148,13 @@ pub(super) fn resolve_orientable_attrs(bargs: &mut BuilderArgs, gtk_widget: &gtk
 
 // concrete widgets
 
-/// @widget combo-box
-fn build_gtk_combo_box(bargs: &mut BuilderArgs) -> Result<gtk::ComboBoxText> {
+/// @widget combo-box-text
+/// @desc A combo box, where one can choose between several items (must be text, combo-box, with other widgets, is not yet implemented)
```
Comment (elkowar) [inline src/widgets/widget_definitions.rs:152]: I don't think it's necessary to mention "not yet implement" stuff in the docs here, as the API of the combo-box-text is quite obviously text-only

```suggestion
/// @desc A combo box allowing the user to choose between several items.
```

**Review by elkowar — CHANGES_REQUESTED** [review]

File: src/widgets/widget_definitions.rs (hunk 2 above)
Comment (alx365) [reply src/widgets/widget_definitions.rs:236]: 🥇

File: src/widgets/widget_definitions.rs (hunk 3 above)
Comment (alx365) [reply src/widgets/widget_definitions.rs:327]: i copied that from the gnome docs. pls no bully 
image

File: src/widgets/widget_definitions.rs (hunk 5 above)
Comment (alx365) [reply src/widgets/widget_definitions.rs:152]: hm tru

**Comment by elkowar** [comment]
Thanks!

**Comment by alx365** [comment]
**EPIC***

**Pull request merged**